packages/orbit-components/src/Itinerary/ItinerarySegment/ItinerarySegmentDetail/index.tsx

```diff
@@ -128,6 +130,20 @@ const StyledIcon = styled.div<{ isFirst?: boolean; isLast?: boolean }>`
   `}
 `;
 
+const StyledItemWrapper = styled.div<{ $offset: number; spaceAfter: SpaceAfterSizes }>`
+  ${({ theme, $offset }) => css`
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    width: calc(100% - ${parseInt(theme.orbit.spaceXSmall, 10) + $offset}px);
```

Comment:
wouldn't it be better to do `width: 100%` here and put `px-xs` here to achieve the desired effect?

Reply:
Unfortunately It is not possible, because offset is calculated for each Itinerary, and with width: 100% it will be overflowing as before on smallMobile :)

Reply:
oh right there's the offset thing. okay